docusaurus/docs/create-a-plugin/extend-a-plugin/fetch-data-from-frontend.md

```diff
@@ -0,0 +1,287 @@
+---
+id: fetch-data-from-frontend
+title: Fetching data from frontend code
+description: Learn how to use the data proxy API to fetch data from frontend code in data source and app plugins in Grafana and overcome CORS limitations
```

Comment:
Data proxy for datasources and plugin proxy for apps 😅 Maybe just proxy or plugin proxy to generalize?

Reply:
Since this doesn't work for panel types I want to be specific on where it works.